src/assets/locales/resources_en_US.json

```diff
@@ -14,5 +14,113 @@
   "skyux_timepicker_input_default_label": {
     "_description": "The label for the timepicker component's input field for screen readers when the consumer has not specified a label.",
     "message": "Time input field."
+  },
+  "skyux_date_range_picker_default_label": {
+    "_description": "The default label for the date range picker.",
```

Comment:
Probably not a big deal, but some of the descriptions have ending punctuation (periods), and others do not...